### PR TITLE
Feature/#12 Create skills field

### PR DESCRIPTION
**Pull request by MangriMen**

### CODE REVIEW

**Comment by MangriMen** [comment]
image

image

**Comment by MangriMen** [comment]
[Create skills field](https://trello.com/c/BJM7GegR/12-create-skills-field)

File: src/components/charlist/Abilities.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+          ability={actualAbility}
+          value={actualValue}
+          enabled={item.enabled}
+        />
+      </>
+    </Box>
+  );
+};
+
+export const Abilities: FC<{
```
Comment (StepanovRoma) [inline src/components/charlist/Abilities.tsx]: The three components should be split into separate files in the same folder

File: src/components/charlist/Charlist.tsx [hunk 2]
```diff
@@ -2,15 +2,49 @@ import { Grid, Typography } from '@mui/material';
...
+      medicine: { enabled: false, value: 0, override: 0 },
+      nature: { enabled: false, value: 0, override: 0 },
+      perception: { enabled: false, value: 0, override: 0 },
+      perfomance: { enabled: false, value: 0, override: 0 },
+      persuasion: { enabled: false, value: 0, override: 0 },
+      religion: { enabled: false, value: 0, override: 0 },
+      sleightOfHand: { enabled: false, value: 0, override: 0 },
+      stealth: { enabled: false, value: 0, override: 0 },
+      survival: { enabled: false, value: 0, override: 0 },
+    },
```
Comment (StepanovRoma) [inline src/components/charlist/Charlist.tsx]: This long thing can be moved to a separate file, or take out the above component

File: src/ducks/data/index.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        for (const lng in state.collected[pkg].translation) {
+          i18n.addResourceBundle(
+            lng,
+            PACKAGES.translationNs,
+            state.collected[pkg].translation[lng],
+            true,
+          );
+        }
+      }
+    },
```
Comment (StepanovRoma) [inline src/ducks/data/index.ts]: It's not a fact that humanity does not yet know how it should be, but personally, I have made data transformations into sagas, and in reducers I just cover the state

**Review by StepanovRoma — CHANGES_REQUESTED** [review]

**Review by FFreestyler — APPROVED** [review]

**Review by StepanovRoma — APPROVED** [review]

**Pull request merged**